pkg/capacityscheduling/elasticquota.go

```diff
@@ -88,15 +90,30 @@ func (e *ElasticQuotaInfo) unreserveResource(request framework.Resource) {
 }
 
 func (e *ElasticQuotaInfo) usedOverMinWith(podRequest *framework.Resource) bool {
-	return cmp2(podRequest, e.Used, e.Min)
+	eqMin := reflect.ValueOf(e).Elem().FieldByName("Min")
+	if eqMin.IsValid() && !eqMin.IsZero() {
+		return cmp2(podRequest, e.Used, e.Min)
+	}
+	// "ElasticQuotaInfo doesn't have Min" means used values exceeded min(0)
+	return true
```

Comment:
Will checking if `e.Min == nil` works too, without using `reflect`? something like:
```go
if e.Min == nil {
    return true
}
return cmp2(podRequest, e.Used, e.Min)
```

Reply:
Thanks for your suggestion. Fixed it.